config/environments/production.rb

```diff
@@ -27,7 +27,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
Try not to live compile your assets:

Reasons: http://stackoverflow.com/questions/8821864/config-assets-compile-true-in-rails-production-why-not